sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -56,6 +47,7 @@
 
     KeyVaultEnvironmentPostProcessorHelper(final ConfigurableEnvironment environment) {
         this.environment = environment;
+        Assert.notNull(environment, "environment must not be null!");
```

Comment:
Do we really need this assertion here?

Reply:
Original code have nonnull check. So I added it, too. Just to keep original logic: https://github.com/Azure/azure-sdk-for-java/pull/12272/files#diff-dcea63cea0abf12a61a1e614cf899b43L204